doc/appendix-de-kassensichv/reference-tables/reference-tables.md

```diff
@@ -279,3 +279,478 @@ For definitions regarding national laws, please refer to the appropriate appendi
+| `ZAHLWAEH_CODE` | Single recordings | Foreign currency code. Only mandatory if foreign currency was used for the payment. |
+| `ZAHLWAEH_BETRAG` | Single recordings | Amount in foreign currency. Only mandatory if foreign currency was used for the payment. |
+| `Z_BUCHUNGSTAG` | Master data | Booking date different from closing date (`Z_ERSTELLUNG`). Only mandatory if the booking date is different to the date of the daily closing receipt. | 
+| Master data for the company, location, terminals, agencies as described in the chapter "Master data module" | Master data | Master data for this cashpoint closing and referenced by the single recordings|
+
+
+The following chapters give you an overview of all DSFinV-K fields, provide you information on how they are filled by ft and how you can send additional data to fill mandatory (listed above) and optional fields that can not be filled by ft.
+
+#### Single recordings module (DE: Einzelaufzeichnungsmodul)
+
+The single recordings provide the basis for data storage. These are divided into two main areas:
+
+- Bonpos (lines.csv)
+- Bonkopf (transactions.csv)
+
+In addition to these two files there are further detail files which are listed in the following. 
+
+##### File: Bonpos (lines.csv)
+
+| **Fieldname**            | **Description**          | **Format**          | **ft.input** |
+|----------------------|--------------------------|---------------------|---------------------|
+| `Z_KASSE_ID` | ID of the (closing) cashpoint | String (50) | `ftCashBoxIdentification` |
+| `Z_ERSTELLUNG` | Date of the cashpoint closing | ISO 8601 und RFC3339 date | from `ftReceiptMoment` of the daily closing receipt |
+| `Z_NR` | Nr. of the cashpoint closing | Integer | Automatically created and filled by ft |
+| `BON_ID` | Action-ID | String (40) | `cbReceiptReference` |
+| `POS_ZEILE` | Line/Position number  | String (50) | `ftChargeItem.Position` if available, otherwise automatically filled by ft |
+| `GUTSCHEIN_NR` | Voucher no.| String (50) | Optional, can be sent via `ftPayItemData` in JSON format. To send, add the key value pair `VoucherNr` e.g. `"ftPayItemData":"{ ..., "VoucherNr":"UAUA91829182HH", ... }"`|
+| `ARTIKELTEXT` | Product/Article text| String (255) | `ftChargeItem.Description` |
+| `POS_TERMINAL_ID` | Terminal-ID of this line (position)| String (50) | `cbTerminalID` |
+| `GV_TYP` | Type of business action  | String (30) | Deducted from `ftChargeItemCase` |
+| `GV_NAME` | Addition to the business action type | String (40) | Optional, can be sent via `ftChargeItemCaseData` in JSON format. To send, add the key value pair `ItemCaseName` e.g. `"ftChargeItemCaseData":"{ ..., "ItemCaseName":"Rabatt - Black Friday", ... }"` |
+| `INHAUS` | Inhouse consumption | 0 or 1 | Optional, can be sent via `ftChargeItemCaseData` in JSON format. To send, add the key value pair `Inhouse` e.g. `"ftChargeItemCaseData":"{ ..., "Inhouse":1, ... }"`, defaults to 0 if not sent, any other value than 0 is interpreted as 1.|
```

Comment:
Should be solved by adding Flags for the `ChargeItem`. Similar to what we do with `ReceiptCaseFlags`, we could just use a logical or to set the Flag. We should take care of documenting this in the `ChargeItemCaseFlags` section